cc1120_arduino/cc1120_arduino.ino

```diff
@@ -31,18 +31,28 @@ void setup() {
     delay(1000);
 
     Serial.println("Starting E2E tests...");
-    bool status = true;
+    cc1120_error_code status;
```

Comment:
This was an issue with the original code too, but if your E2E tests fail in this for loop, the status variable is not reset once you restart the tests - need to move this code into the for loop